packages/forklift-console-plugin/src/modules/NetworkMaps/__tests__/mergeData.test.ts

```diff
@@ -16,9 +17,9 @@ describe('merging network data', () => {
     expect(mergeNetworkData([], [])).toHaveLength(0);
   });
 
-  test('standard mock data', () => {
-    const mappings = MOCK_NETWORK_MAPPINGS as NetworkMapResource[];
-    const providers = MOCK_CLUSTER_PROVIDERS as V1beta1Provider[];
+  test('with legacy mock data', () => {
+    const mappings = LEGACY_MOCK_NETWORK_MAPPINGS as NetworkMapResource[];
```

Comment:
use the get data method instead of direct import of the const,
the idea is to have the same data in the server (web-worker) and the test (mock data)

Reply:
I've tried but it looks awkward - the current unit test are written on different level with no reference to network/queries/paths

Reply:
```
   test('with new mock data', () => {
-    const mappings = MOCK_NETWORK_MAPPINGS as NetworkMapResource[];
-    const providers = MOCK_CLUSTER_PROVIDERS as V1beta1Provider[];
+    const mappings = getMockData({
+      pathname: '/api/proxy/plugin/forklift-console-plugin/forklift-inventory/providers',
+    })?.body as NetworkMapResource[];
+
+    const providers = getMockData({
+      pathname: 'api/kubernetes/apis/forklift.konveyor.io/v1beta1/providers',
+    })?.body as V1beta1Provider[];
     const merged = mergeNetworkData(mappings, providers)
```

Reply:
ok, we will do that in a followup, more carefully